bedrock/products/templates/products/vpn/includes/pricing-fixed.html

```diff
@@ -0,0 +1,50 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Note for reviewer: this file also isn't included in the page yet. I'm adding it here as placeholder so we can start lcoalizing the strings needed for https://github.com/mozilla/bedrock/issues/9948